README.md

```diff
@@ -193,6 +197,26 @@ By default, when printing the help message only a selection of attributes are pr
 nextflow run my_pipeline --help param_name
 ```
 
+### Validate an input file provided by params with another JSON schema
+
+By provided the `schema` field in one off the parameters, the function will automatically validate the provided file using this JSON schema. It can validate CSV, TSV and simple YAML files.
+The path of the schema file must be relative to the root of the pipeline directory. See an example in the `input` field from the [example schema.json](https://raw.githubusercontent.com/nextflow-io/nf-validation/master/plugins/nf-validation/src/testResources/nextflow_schema_with_samplesheet.json#L20).
```

Comment:
I like this, but won't this cause problems for pipelines that can use multiple kinds of samplesheets?

Reply:
What kinds of sample sheets are you thinking of? If they can't be validated by a JSON schema, this `schema` field is not required.

Reply:
Ah I think I get it now, you mean when there are multiple sample sheets that should be validated by different schemas?

Reply:
I meant pipelines that can have multiple different kinds of input (e.g. Sarek can take fastq, bams, vcfs...). Those are different samplesheets I think, or would you suggest to make one big schema that works for all the different samplesheets (which is possible I think)

Reply:
(didn't see you post that second comment, but yes exactly that)

Reply:
I see, Sarek is always the edge case 😂 I would suggest having a schema which can validate all sample sheets or use different parameters to provide them.

Reply:
True that seems the best solution :)